extensions/reactive-messaging-http/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -10,4 +10,9 @@ metadata:
     - "web"
     - "reactive"
     - "messaging"
-  status: "experimental"
\ No newline at end of file
+  status: "experimental"
```

Comment:
There is no provided codestart for HTTP so why adding it here ?

Reply:
Good point!